examples/java/src/main/java/org/apache/beam/examples/snippets/Snippets.java

```diff
@@ -641,7 +643,7 @@ public void process(
 
                       @ProcessElement
                       public void process(ProcessContext c, @Timestamp Instant timestamp) {
-                        Map<String, String> keyMap = c.sideInput(map);
+                        Map<String, String> keyMap = c.sideInput(mapIterable[0]);
```

Comment:
Maybe add a comment here. Better behavior might be using the last element of that array.

Reply:
Added a comment

> Better behavior might be using the last element of that array.

In general, it shouldn't make much of a difference, and the ordering isn't deterministic AFAIK. I'd vote to leave it as is since it makes the example simpler.